grafana_objects/client_grafana_objects_create_update_test.go

```diff
@@ -0,0 +1,112 @@
+package grafana_objects_test
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"testing"
+
+	"github.com/logzio/logzio_terraform_client/grafana_objects"
+	"github.com/stretchr/testify/assert"
+)
+
+func createUpdateMockHandler(t *testing.T) func(http.ResponseWriter, *http.Request) {
```

Comment:
Please refactor the test like in here:
https://github.com/logzio/logzio_terraform_client/blob/master/archive_logs/archive_logs_setup_test.go

Reply:
hi @mirii1994 , thanks for your review, do you need me to do it for get_by_uid and delete_by_uid too?